dart/collision/fcl/FCLCollisionNode.h

```diff
@@ -62,12 +63,10 @@ class FCLCollisionNode;
 struct FCLUserData
```

Comment:
Does this struct need to be exposed to the user? Maybe we should have it hidden in ``FCLCollisionNode.h``.